packages/emoji-mart/src/components/Navigation/Navigation.tsx

```diff
@@ -54,7 +54,12 @@ export default class Navigation extends PureComponent {
     let selectedCategoryIndex = null
 
     return (
-      <nav id="nav" class="padding" data-position={this.props.position}>
+      <nav
+        id="nav"
+        class="padding"
+        data-position={this.props.position}
+        dir={this.props.dir}
```

Comment:
Should be `this.dir`

Reply:
Are you sure? in this component we don't have `this.dir` and I sending `dir` as a props from Picker to Navigation

Reply:
Oh, my bad! I thought this was in Picker